rust/src/wrapper/device/mod.rs

```diff
@@ -112,51 +116,73 @@ impl Device {
         Python::with_gil(|py| {
             let kwargs = PyDict::new(py);
             kwargs.set_opt_item("name", name)?;
-            kwargs.set_opt_item("address", address)?;
+            kwargs.set_opt_item("address", address.map(|a| a.try_to_py(py)).transpose()?)?;
             kwargs.set_opt_item("config", config)?;
             kwargs.set_opt_item("host", host)?;
             kwargs.set_opt_item("generic_access_service", generic_access_service)?;
 
-            PyModule::import(py, intern!(py, "bumble.device"))?
-                .getattr(intern!(py, "Device"))?
+            let device_ctor = PyModule::import(py, intern!(py, "bumble.device"))?
+                .getattr(intern!(py, "Device"))?;
+
+            // Needed for Python 3.8-3.9, in which the Semaphore object, when constructed, calls
+            // `get_event_loop`.
+            wrap_python_async(py, device_ctor)?
```

Comment:
Had to sprinkle this on Device and Peer invocations that end up instantiating an `asyncio.Semaphore`